# Description: Add "-lapr-1 -lsvn_subr-1" to the linker options.
#  Without this patch the package will FTBFS with binutils-gold.
# Forwarded: http://gitorious.org/svn2git/svn2git/merge_requests/5
# Author: José Manuel Santamaría Lema <panfaust@gmail.com>
--- svn-all-fast-export-0~git20100701.orig/src/src.pro
+++ svn-all-fast-export-0~git20100701/src/src.pro
@@ -13,7 +13,7 @@
 
 INCLUDEPATH += . $$SVN_INCLUDE $$APR_INCLUDE
 !isEmpty($$SVN_LIBDIR): LIBS += -L$$SVN_LIBDIR
-LIBS += -lsvn_fs-1 -lsvn_repos-1
+LIBS += -lsvn_fs-1 -lsvn_repos-1 -lapr-1 -lsvn_subr-1
 
 # Input
 SOURCES += ruleparser.cpp \
